src/core/exchange/exchange-selectors.js

```diff
@@ -78,6 +78,31 @@ function searchRoutes(
   }
 }
 
+function isFiatCode(currencyCode: string): boolean {
+  return /^iso:/.test(currencyCode)
+}
+
+/**
+ * Create routes to USD if no deep routes to fiat exist. Helpful for hard
+ * coded crypto/crypto exchange rates.
+ */
+function createDeepFiatRoutes(
+  search: ExchangeSearch,
+  fromCurrency: string
+): void {
+  const newHints = []
+  for (const fromRoute of Object.keys(search.routes[fromCurrency])) {
+    for (const deepRoute of Object.keys(search.routes[fromRoute])) {
+      if (isFiatCode(deepRoute)) {
+        return
+      } else {
```

Comment:
The `return; } else {` pattern isn't needed, since the `return` statement prevents the next line from running anyhow. Therefore, we can do:

```js
if (isFiatCode(deepRoute)) return
newHints.push({ fromCurrency: fromRoute, toCurrency: 'iso:USD' })
```

Since this is just a style change, feel free to toss it in when you squash away the `fixup!` commit.